src/main.tsx

```diff
@@ -1,10 +1,6 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.tsx'
-import './index.css'
+import ReactDOM from "react-dom/client";
+import "./css/index.css";
 
-ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-)
+ReactDOM.createRoot(document.getElementById("root")!).render(
+  <h1>Tahsin</h1>
```

Comment:
haven't rendered the App Component here
# make suire to change them 
the previews are the text only that was on the inital setup now it i needs to render the App